src/RamTransforms.cpp

```diff
@@ -30,6 +30,42 @@
 
 namespace souffle {
 
+bool ExpandFilterTransformer::expandFilters(RamProgram& program) {
+    // flag to determine whether the RAM program has changed
+    bool changed = false;
+
+    std::vector<std::unique_ptr<RamCondition>> conditionList;
```

Comment:
Does this need to be in this scope? It only seems to be used within the filterRewriter.

Reply:
Yeah doesn't need to be